### PR TITLE
Add bandpass information into DonutStamp objects.

### PR DESCRIPTION
**Pull request by jbkalmbach**

### CODE REVIEW

File: python/lsst/ts/wep/task/CutOutDonutsBase.py [hunk 1]
```diff
@@ -381,6 +381,7 @@ def cutOutStamps(self, exposure, donutCatalog, defocalType, cameraName):
         detectorType = exposure.getDetector().getType()
         pixelScale = exposure.getWcs().getPixelScale().asArcseconds()
         camType = getCamTypeFromButlerName(cameraName, detectorType)
+        bandpass = exposure.filter.bandLabel
         # Get template
```
Comment (suberlak) [inline python/lsst/ts/wep/task/CutOutDonutsBase.py:385]: Is `bandLabel` property something that has been around for some time, or was it introduced in some recent weekly? Just making sure it doesn't break when trying to run on an older dataset.

File: python/lsst/ts/wep/task/DonutStamp.py [hunk 2]
```diff
@@ -61,6 +61,8 @@ class DonutStamp(AbstractStamp):
     cam_name : `str`
         Camera name for the stamp image. "LSSTCam" or "LSSTComCam"
         are available camera names currently.
+    bandpass : `str`
+        The bandpass for the stamp image.
```
Comment (suberlak) [inline python/lsst/ts/wep/task/DonutStamp.py:65]: Are there any allowed/default values?

**Review by suberlak — APPROVED** [review]
Thanks for the update!

File: python/lsst/ts/wep/task/CutOutDonutsBase.py (hunk 1 above)
Comment (jbkalmbach) [reply python/lsst/ts/wep/task/CutOutDonutsBase.py:385]: A couple years old (looks like w_2020_50) so hopefully that shouldn't break anything you're currently using?

Comment (jbkalmbach) [reply python/lsst/ts/wep/task/CutOutDonutsBase.py:385]: https://community.lsst.org/t/afw-image-filter-is-now-deprecated/4675

File: python/lsst/ts/wep/task/DonutStamp.py (hunk 2 above)
Comment (jbkalmbach) [reply python/lsst/ts/wep/task/DonutStamp.py:65]: Ah, good point. This is not optional but I will add information below in the factory method pointing out the default value used when loading stamps created with an older version.

**Pull request merged**